### ISSUE TITLE
kubelet-NodeConformance tests are broken

### ISSUE THREAD

**Issue by SergeyKanzhelev** [kubernetes/test-infra#22250]
**What happened**:

Tests are broken: https://testgrid.k8s.io/sig-node-critical#kubelet-NodeConformance Wouldn't even start tests.

Same tests with containerd pass: https://testgrid.k8s.io/sig-node-critical#containerd-NodeConformance

**What you expected to happen**:

Tests pass.

/sig node

**Comment by BenTheElder** [comment]
> Tests are broken: https://testgrid.k8s.io/sig-node-critical#kubelet-NodeConformance Wouldn't even start tests.

Is there any reason to suspect this is a bug in test-infra and not a bug in kubernetes / node_e2e / the cluster-up scripts ...?

Failing tests are rarely a test-infra bug.

**Comment by ehashman** [comment]
@SergeyKanzhelev this is a duplicate of https://github.com/kubernetes/test-infra/issues/18973 and https://github.com/kubernetes/kubernetes/issues/97130 no?

**Comment by ehashman** [comment]
I have a PR up to just remove these tests as they appear to be duplicates: https://github.com/kubernetes/test-infra/pull/22454

**Comment by ehashman** [comment]
Example of one of the failing jobs: https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet-conformance/1402940005772431360

**Comment by odinuge** [comment]
Looks like the test is just timing out, with a timeout of 65 min. Not sure what tests are running under this test suite, but it would be interesting to see what is the root cause to this before we remove it (it might be some bug or something). If we get https://github.com/kubernetes/kubernetes/pull/102209 merged, we would be able to debug why it fails.

**Comment by dims** [comment]
@odinuge i've approve/lgtm'ed https://github.com/kubernetes/kubernetes/pull/102209 please feel free to remove hold when you are ready.

**Comment by odinuge** [comment]
A run with proper output and artifacts can be found here: https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet-conformance/1405056909827903488

It simply looks like there are too many tests to finish serially on a single node with a timeout of `65min`. The `ci-containerd-node-e2e` job, the "duplicate", runs with `8` ginkgo-nodes compared to `1` for the `ci-kubernetes-node-kubelet-conformance` job (aka. 8 tests in parallel, instead of serially).

https://github.com/kubernetes/test-infra/blob/a5528997fea7e64a44a3335ad95c4d3df6514ed7/config/jobs/kubernetes/sig-node/containerd.yaml#L218 

I am fine with dropping the docker version now that we know what is the issue is. (Or should we try to bump `--nodes` instead?)

**Comment by odinuge** [comment]
There is however a small difference between the two. The docker one will not skip `[Serial]`, while the containerd one will.

I cannot find any `[NodeConformance]` tests with `[Serial]`, but that would be changed when https://github.com/kubernetes/kubernetes/pull/102774 is merged I assume. So, the containerd version will skip that test as it is now.

/cc @bart0sh

**Comment by ehashman** [comment]
> There is however a small difference between the two. The docker one will not skip `[Serial]`, while the containerd one will.

Yeah, I saw that as well, and noted it when I submitted https://github.com/kubernetes/test-infra/pull/22454#issue-663787438

> I cannot find any `[NodeConformance]` tests with `[Serial]`, but that would be changed when [kubernetes/kubernetes#102774](https://github.com/kubernetes/kubernetes/pull/102774) is merged I assume. So, the containerd version will skip that test as it is now.

We discussed this in the CI subproject meeting today and we're pretty sure that it shouldn't be getting a NodeConformance label (https://github.com/kubernetes/kubernetes/pull/102774#issuecomment-862579913).

**Comment by ehashman** [comment]
/close

I removed the test job.
